### PR TITLE
feat: SEO Optimise site description

### PR DESCRIPTION
**Pull request by xxori**
### Description
Including more keywords in meta description helps seo

### Changes Made
updated meta and og description in index.html

### Related Issues
*Reference any related issues using keywords like "Fixes #issue-number".*

### Additional Notes
*Add any other notes or screenshots that might be helpful for the reviewers.*

### CODE REVIEW

**Comment by xxori** [comment]
# 🥺

**Review by jsun969 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
 		<title>MyTimetable</title>
 		<meta
 			name="description"
-			content="A drag-and-drop timetable planner for University of Adelaide students."
+			content="MyTimetable is a simple drag-and-drop timetable planner for University of Adelaide students."
```
Comment (phoenixpereira) [inline index.html:12]: The meta description should be about 150-220 characters, so maybe something like "MyTimetable is a simple drag-and-drop timetable planner for University of Adelaide students. Easily organise your course classes and create the perfect timetable with this tool by the CS Club Open Source Team."

**Review by phoenixpereira — CHANGES_REQUESTED** [review]

**Review by rayokamoto — APPROVED** [review]
LGTM

File: index.html [hunk 2]
```diff
@@ -19,7 +19,7 @@
 		<meta property="og:url" content="https://mytimetable.csclub.org.au/" />
 		<meta
 			property="og:description"
-			content="A drag-and-drop timetable planner for University of Adelaide students."
+			content="MyTimetable is a simple drag-and-drop timetable planner for University of Adelaide students. Easily organise your course classes and create the perfect timetable with this tool by the CS Club Open Source Team."
```
Comment (phoenixpereira) [inline index.html:22]: The longer description was just needed for the top meta description, the og and twitter descriptions have lower character limits, so make them just use the first sentence.

**Review by phoenixpereira — CHANGES_REQUESTED** [review]

**Review by phoenixpereira — APPROVED** [review]

**Pull request merged**